docs/CONNECTOR_PROTOCOL.md

```diff
@@ -165,23 +116,21 @@ This is our main communication index, used to communicate the connector's config
 }
 ```
 
-### Jobs index
-In addition to the .elastic-connectors index for documents, we have an index to log all jobs run by connectors. This is the .elastic-connectors-sync-jobs index. Each JSON document will have the following structure:
+#### Connector job index `.elastic-connectors-sync-jobs`
+In addition to the connector index `.elastic-connectors`, we have an additional index to log all jobs run by connectors. This is the `.elastic-connectors-sync-jobs` index. Each JSON document will have the following structure:
 ```
 {
-  connector: object; -> Copy of the current connector document
```

Comment:
Why is this connector object removed? I know that current connectors aren't generally writing that connector object, but this is something they should start doing.

Reply:
replied in another comment.